css/components/badges.css

```diff
@@ -1,99 +1,99 @@
 .badge-wrapper {
-	min-width: 2rem;
-	padding: 1rem;
-	font-size: 1.5rem;
+  min-width: 2rem;
+  padding: 1rem;
+  font-size: 1.5rem;
 }
 
 .badge {
-	position: relative;
+  position: relative;
 }
 
 .badge-counter {
-	position: absolute;
-	font-size: 0.8rem;
-	min-width: 1.5rem;
-	min-height: 1.5rem;
-	display: flex;
-	align-items: center;
-	justify-content: center;
+  position: absolute;
+  font-size: 0.8rem;
+  min-width: 1.5rem;
+  min-height: 1.5rem;
```

Comment:
just a suggestion here, I used `em` as unit for the width and height of the badges or basically any icon which must be a complete circle and has only one text inside it. Since, `em` is relative to the font size of the element, it will always be round. like here, you could do `min-width: 2em; min-height: 2em;` then you could easily change the size of the badge by simply changing the font size and it would still be a circle.

Reply:
made changes